index.js

```diff
@@ -4,9 +4,11 @@ module.exports = {
   },
   "extends": "eslint:recommended",
   "rules": {
-    "prefer-const": "warn",
-    "no-var": "warn",
-    "prefer-template": "warn",
+    "prefer-const": "error",
+    "no-var": "error",
+    "prefer-template": "error",
+    "max-params": ["error", 3],
```

Comment:
just a thought, personal opinion (following your message in slack):
explicit arguments are better than passing object as it's not clear what function needs looking at function/method signature. being object (as no type enforcement) it can contain anything and function will just keep growing. 

having 3 or more function _may be_ a symptom that this function is doing too many things.